src/ProjectTemplates/Web.ProjectTemplates/content/WebApiAot-CSharp/Program.cs

```diff
@@ -9,7 +9,8 @@
 
 var app = builder.Build();
 
-var sampleTodos = new Todo[] {
+var sampleTodos = new Todo[]
```

Comment:
```suggestion
Todo[] sampleTodos = [
```

Small nit unrelated to your contribution but while we're here perhaps we should update this to use collection expressions?